### DISCUSSION TITLE
SaveSeuratRds vs saveRDS

### DISCUSSION THREAD

**Discussion by royfrancis** [satijalab/seurat#8276] [Q&A]
What is the difference between `SaveSeuratRds()` vs `saveRDS()`? Is there a difference in how they are used with Seurat v4, v5 and v5 with Sketch objects?

**Comment by hdionne** [comment]
The code for SaveSeuratRDS() has seems to use saveRDS itself in it, but there there's an additional ~100 lines of code.
The function itself starts at line ~600 of https://github.com/satijalab/seurat-object/blob/master/R/seurat.R
I'm not sure what the code in between does for sure though.
